leasing/enums.py

```diff
@@ -487,3 +487,17 @@ class Labels:
         ADDITIONAL_YARD = pgettext_lazy('Basis of rent type', 'Additional yard')
         FIELD = pgettext_lazy('Basis of rent type', 'Field')
         MAST = pgettext_lazy('Basis of rent type', 'Mast')
+
+
+class Zone(Enum):
+    """
+    In Finnish: Vyöhyke
+    """
+    ZONE_1 = 'zone 1'
```

Comment:
(I think I added this comment already, but...)

In enums I've used the kind of value that doesn't need escaping in URLs etc. Like in every other enum the spaces are replaced with underscores.